src/main/scala/org/scalacheck/util/Buildable.scala

```diff
@@ -22,33 +22,14 @@ trait Buildable[T,C] extends Serializable {
 
 object Buildable extends BuildableVersionSpecific {
   import java.util.ArrayList
-  implicit def buildableArrayList[T]: Buildable[T, ArrayList[T]] = new Buildable[T,ArrayList[T]] {
-    def builder = new ArrayListBuilder[T]
-  }
-}
-
-/*
-object Buildable2 {
-
-  implicit def buildableMutableMap[T,U] = new Buildable2[T,U,mutable.Map] {
-    def builder = mutable.Map.newBuilder
-  }
-
-  implicit def buildableImmutableMap[T,U] = new Buildable2[T,U,immutable.Map] {
-    def builder = immutable.Map.newBuilder
-  }
-
-  implicit def buildableMap[T,U] = new Buildable2[T,U,Map] {
-    def builder = Map.newBuilder
-  }
-
-  implicit def buildableImmutableSortedMap[T: Ordering, U] = new Buildable2[T,U,immutable.SortedMap] {
-    def builder = immutable.SortedMap.newBuilder
-  }
-
-  implicit def buildableSortedMap[T: Ordering, U] = new Buildable2[T,U,SortedMap] {
-    def builder = SortedMap.newBuilder
-  }
-
+  implicit def buildableArrayList[T]: Buildable[T, ArrayList[T]] =
+    new Buildable[T, ArrayList[T]] {
+      def builder = new ArrayListBuilder[T]
+    }
+
+  implicit def buildableSeq[T]: Buildable[T, Seq[T]] =
```

Comment:
can't we do something like `implicit def canBuild[A, B](implicit cbf: CanBuildFrom[Nothing, A, B]): Buildable[A, B]` and let CanBuildFrom do the work?

Reply:
Maybe? I just chose to do the simplest thing in line with the existing code. It's worth experimenting with.

Reply:
After looking into this for a second I think the 2.13 collections makes relying on `CanBuildFrom` for this a bit dicey. We only need to support `Seq` and this is really only for internal use so I'd rather not try to do something more general until we need it.